app/uploaders/image_uploader.rb

```diff
@@ -1,22 +1,58 @@
 class ImageUploader < WhitehallUploader
   include CarrierWave::MiniMagick
 
+  process :store_dimensions
+
   configure do |config|
     config.remove_previously_stored_files_after_update = false
     config.storage = Storage::PreviewableStorage
   end
 
+  def downloader
```

Comment:
Where is this called?

Reply:
This is [overloading the base](https://www.rubydoc.info/gems/carrierwave/CarrierWave/Downloader/Base) `downloader` from `CarrierWave::Uploader::Base`. We need to override [skip_ssrf_protection](https://github.com/alphagov/whitehall/blob/a9ca8ed1a0b188a2b4ec9d6e59f63a40cb1c4bc3/app/uploaders/whitehall_downloader.rb), without doing this an error will occur [when EditionImagesController calls](https://github.com/alphagov/whitehall/blob/a9ca8ed1a0b188a2b4ec9d6e59f63a40cb1c4bc3/app/controllers/admin/edition_images_controller.rb#L26) `download!` in test and development environments.

Reply:
I can put a comment saying this in the `image_uploader` if that makes it clearer.